src/backend/app.js

```diff
@@ -21,6 +24,76 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+
+// Route to get future meals sorted by ID
+app.get("/future-meals", async (req, res) => {
+  try {
+    const meals = await knex
+      .select()
+      .from("meal")
+      .where("when", ">", knex.fn.now());
+      console.log(meals)
+    res.status(200).json(meals);
+  } catch (error) {
+    res.status(500).json("500. Internal Server Error");
+  }
+});
+
+
+// Respond with all meals in the past (relative to the when datetime)
+app.get("/past-meals", async (req, res) => {
+  try {
+    const meals = await knex
+      .select()
+      .from("meal")
+      .where("when", "<", knex.fn.now())
+      .orderBy("id");
+    res.status(200).json(meals);
+  } catch (error) {
+    res.status(500).json("500. Internal Server Error");
+  }
+});
+// Respond with all meals sorted by ID
+app.get("/all-meals", async (req, res) => {
+  try {
+    const meals = await knex
+      .select("title", "description")
+      .from("meal")
+      .orderBy("id");
+    res.status(200).json(meals);
+  } catch (error) {
+    res.status(500).json("500. Internal Server Error");
+  }
+});
+
+//Respond with the first meal (meaning with the minimum id)
+app.get("/first-meal", async (req, res) => {
+  try {
+    const meal = await knex
+      .select("title", "description")
+      .from("meal")
+      .orderBy("id")
+      .first();
+    res.status(200).json(meal);
```

Comment:
add an `if` condition for the response. If meal doesn't exist it should respond with a `404` response with a message, that the meal not found or something. And if exist it should return meal.